### PR TITLE
Update Model Builder install instructions

### LINKED ISSUE #24836: Update How to install Model Builder
- Update pre-reqs
- Remove note about .NET Core 3.0 SDK not being supported
- Update install instructions to https://dotnet.microsoft.com/learn/ml-dotnet/get-started-tutorial/install
- Review all other outdated content and update as necessary

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5692cfb5-52e1-b6b9-2b87-7ea0decc2882
* Version Independent ID: 2da762a6-5bc4-a8d1-afae-f1a8243eee33
* Content: [How to install Model Builder - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder)
* Content Source: [docs/machine-learning/how-to-guides/install-model-builder.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/how-to-guides/install-model-builder.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by jwood803**
Update Model Builder install instructions.

- Remove references to VS 2017
- Update prerequisites
- Update how to install Model Builder

Fixes #24836

[Preview](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder?branch=pr-en-us-24978)

AB#1864081

### CODE REVIEW

File: docs/machine-learning/how-to-guides/install-model-builder.md [hunk 1]
```diff
@@ -17,11 +17,8 @@ Learn how to install ML.NET Model Builder to add machine learning to your .NET a
 
 ## Prerequisites
 
-- Visual Studio 2017 version 15.9.12 or later / Visual Studio 2019
-- .NET Core 2.1 SDK or later.
-
-> [!NOTE]
-> .NET Core 3.0 SDK is not currently supported.
+- Visual Studio 2019
+- .NET Core 3.0 SDK or later.
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/install-model-builder.md:21]: .NET Core 3.0 is out of support. I don't think it should be mentioned here.

**Comment by luisquintanilla** [comment]
@jwood803 for limitations, only 1 & 4 are still valid.

image

**Comment by Youssef1313** [comment]
@jwood803 In case you can't find the warnings, here they are:

```
Line 43: [Warning] Alt text 'Enable Model Builder' is duplicated. Within an article, alt text must be unique.
Line 51: [Warning] Alt text 'Enable Model Builder' is duplicated. Within an article, alt text must be unique.
```

**Comment by jwood803** [comment]
@Youssef1313 Thanks! I couldn't find those in the build.

**Review by luisquintanilla — APPROVED** [review]
LGTM

**Pull request merged**